webapp/webpack.config.js

```diff
@@ -36,6 +36,9 @@ const config = {
     resolve: {
         alias: {
             '@': path.resolve(__dirname, 'src'),
+            // temporary fix for `mattermost-redux` v10+
+            // see more: https://github.com/mattermost/mattermost/issues/31060
+            'mattermost-redux': 'node_modules/mattermost-redux/lib',
```

Comment:
Good fix! I suppose it'll be a bit less temporary if anyone uses this template in the time before it's removed, but that should be fine since this is at worst just redundant

Reply:
We are going to fix https://github.com/mattermost/mattermost/issues/31060 internally so this won't be needed. I'm going to hold off approving this as I'd like to wait for the internal fix